style.css

```diff
@@ -0,0 +1,92 @@
+@media screen and (max-width: 375px) {
```

Comment:
- [ ] You should note use media queries for now since this is mobile first approach